index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css"/>
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="container">
+  
+    <header class="header">
+      <ul>
+        <li><img src="./assets/Time.png" alt="clock" class="top"></li>
+        <li><img src="./assets/Group 3.png" alt="bars" class="top"></li>
+      </ul>
```

Comment:
- Hey, please kindly remove this part as they are not part of the design as the design was been viewed on a mobile device, that's you could see in the design.